CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2018.02.0
+* But - Fixed an issue where container wasn't able to talk to agent endpoint.
```

Comment:
spelling: `Bug`. You should also add an entry to attribute to https://github.com/aws/amazon-ecs-cni-plugins/pull/59